content/techniques/configuration.md

```diff
@@ -115,6 +115,20 @@ export class AppService {
 
 > info **Hint** Instead of importing `ConfigModule` in each module, you can alternatively declare `ConfigModule` as a [global module](https://docs.nestjs.com/modules#global-modules).
 
+#### Using the Config Service in `main.ts`
+While our config is a stored in a service, it can still be used in the main.ts file.  
+This way, you can use it to store variables such as the application port or the CORS host.
+
+To access it, you must use `app.get`, followed by the name of your service:
+```typescript
+const config = app.get("ConfigService");
```

Comment:
Why didn't you use `ConfigService`? (instead of the `"ConfigService"` string)

Reply:
I didn't know we could do this actually

I'll update the PR with the config service directly

Reply:
Ping

Reply:
Updated! Sorry for the delay